paper_framework_sources/p2079_system_execution_context.bs

```diff
@@ -397,24 +378,20 @@ public:
         on which `start` has been called, but are not yet running (and are hence not yet guaranteed to make progress) **must**
         complete with `set_stopped` as soon as is practical.
     - if `snd` is connected with a `receiver`, and the resulting operation is started but the implementation reaches an error condition, the operation must complete with `set_error(ep)`, where `ep` is of type `std::exception_ptr`.
-  - The `bulk` algorithm is customized for the scheduler returned by `get_system_scheduler()`
+  - The `bulk` algorithm is customized for the scheduler returned by `get_parallel_scheduler()`
     - the corresponding sender has the same properties as the sender returned by `schedule()`;
     - the functor given to `bulk` is invoked from threads belonging to the system execution context.
 
 ## Replaceability API ## {#replaceability_api}
 
 <pre highlight="c++">
 namespace std::execution::system_context_replaceability {
+  // Called by the frontend.
+  // Users might replace instantiations.
```

Comment:
Use of the word "instantiations" here is not clear.

Reply:
I think it means "Users may replace particular template specializations"

Reply:
Removed the part with the template after getting some feedback from Ruslan & Jens.